### PR TITLE
Prettier html

### PR DESCRIPTION
**Pull request by sfdctaka**

### CODE REVIEW

**Comment by sfdctaka** [comment]
If approved I will merge this after @dbreese 's code generation PR is merged.

File: package.json
Comment (khawkins) [inline package.json:55]: Nit: shouldn't the `--list-different` and `--write` arguments look basically the same? Specifically, should we pick one or the other of `\"src/**/*.{ts, js}\"` and `\"{src, resources/instructions}/**/*.{ts, js}\"`, for consistency?

(And I'd lobby for the former, just because it's easier to read.)

**Review by khawkins — APPROVED** [review]

**Pull request merged**
